apps/cricket-score-frontend/README.md

```
@@ -154,23 +156,25 @@
 > **⚠️ Important**: Never commit your `.env.local` file to version control. The actual values should be kept secure.
 
 #### 🔗 About the Backend API
-This frontend connects to the [Cricket Score API](https://github.com/deveshsangwan/cricketScoreApi) backend. You can:
+This frontend connects to the backend inside this monorepo at `apps/cricketscoreapi`. You can:
 - **Use the deployed API** at `https://cricketapi-r9zv.onrender.com` (default)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)